jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/resources/org/jbpm/workbench/cm/client/resources/i18n/Constants.properties

```diff
@@ -21,6 +21,10 @@ CaseDetails=Case Details
 Owner=Owner
 Roles=Roles
 RoleAssignments=Role Assignments
+DeleteAssignment=Delete assignment
```

Comment:
I think this can be discussed but perhaps we should use _Remove_ instead of _Delete_.